tensorflow/contrib/layers/ln_kernels/layer_norm_fused_grad_op.cc

```diff
@@ -0,0 +1,356 @@
+/* Copyright 2015 The TensorFlow Authors. All Rights Reserved.
```

Comment:
I see, I'll find an example tomorrow. Basically, there are kernels and ops in  `tensorflow/core/kernels`, you can see how they are built.